Mathlib/Algebra/Order/Floor.lean

```diff
@@ -1340,6 +1341,16 @@ theorem round_le (x : α) (z : ℤ) : |x - round x| ≤ |x - z| := by
     norm_cast
     exact floor_le_sub_one_iff.mpr hx
 
+@[simp]
+theorem round_half_bot [HSub α ℝ α] : x - (1 / 2 : ℝ) < round x := by
+  rw [round_eq x, show (x - 1 / 2 : ℝ) = (x + 1 / 2) - 1 by ring]
```

Comment:
```suggestion
@[simp]
theorem round_half_bot : x - (1 / 2 : ℚ) < round x := by
  rw [round_eq x, show x - (1 / 2 : ℚ) = x + 1 / 2 - 1 by ring]
```
You shouldn't use `HSub α ℝ α`, the subtraction it defines is unconnected to `α`'s field structure, you should use the coercion from `ℚ` to `α` (which is part of `α`'s field structure).
Additionally I'm not sure it should be a `simp` lemma. Is there any reason for that?